src/gurt/dlog.c

```diff
@@ -591,11 +763,19 @@ d_log_open(char *tag, int maxfac_hint, int default_mask, int stderr_mask,
 	int	truncate = 0, rc;
 	char	*env;
 	char	*buffer = NULL;
+	uint64_t log_size = LOG_SIZE_DEF;
 
 	env = getenv(D_LOG_TRUNCATE_ENV);
 	if (env != NULL && atoi(env) > 0)
 		truncate = 1;
 
+	env = getenv(D_LOG_SIZE_ENV);
+	if (env != NULL) {
```

Comment:
we could probably add a function like d_getenv_size that takes into account K, M, and G like ts_val_factor in daos_perf.c and make this a little more friendly to set (e.g. 1G rather than 1000000000 or whatever).